tritium-metrics/src/main/metrics/invocation-metrics.yml

```diff
@@ -0,0 +1,15 @@
+options:
+  javaPackage: com.palantir.tritium.event.metrics
+  javaVisibility: packagePrivate
+namespaces:
+  invocation:
+    docs: Tritium invocation proxy metrics for components wrapped using `Tritium.instrument` or `Instrumentation.builder()` with tagged metrics handlers.
+    metrics:
+      success:
+        type: timer
+        tags: [service-name, method-name]
```

Comment:
we'll have to eat cost migrating from `endpoint` to `method-name` until everything is migrated, and we'll need to kill off the old ones to avoid long tail cost of duplicate metrics

Reply:
The most difficult part is discovering places that rely on the existing metrics. `service-name` and `endpoint` tags match those used by our per-endpoint server metrics, and the names are completely dynamic based on the interface name.
I'm a bit worried about keeping both tagged metric handlers around for too long because they're the heaviest handlers, and timers are expensive. I think an announcement email is the way to go once we've allowed a version to percolate out with the new metrics before we stop wiring the old metrics by default.